config/samples/base/openstackcontrolplane/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -205,7 +205,7 @@ spec:
       designateProducer:
         replicas: 1 # backend needs to be configured
       designateMdns:
-        replicas: 0 # backend needs to be configured
+        replicas: 1 # backend needs to be configured
```

Comment:
comment here says backend needs to be configured still. If you are setting replicas to 1 does this (and if functionally works) does this comment still make sense?

Reply:
It looks like it is a generic comment that appear on many files. We are still configuring MiniDNS backend on the designate-operator repository, so I think it still applies.
This openstack-operator patch mainly saves us time, so we won't have to modify the csv manually with the same values. We modify its value to make sure changes in the current replica count will behave correctly with our changes.